modules/boringssl/0.0.0-20240126-22d349c/MODULE.bazel

```diff
@@ -0,0 +1,7 @@
+module(
+    name = "boringssl",
+    version = "0.0.0-20240126-22d349c",
+    compatibility_level = 29,  # BORINGSSL_API_VERSION from src/include/openssl/base.h#L112
```

Comment:
I would rather keep the compatibility level constant as BoringSSL encourages using the preprocessor macro to support multiple versions at the same time: https://github.com/google/boringssl/blob/db7308de87ea138e7bbcbbb00dfc9b841774ba2f/include/openssl/base.h#L109

Bumping the compatibility level would make this more difficult by pushing version handling onto the Bazel level.